pages/clustering/high-availability.mdx

```diff
@@ -178,6 +179,18 @@ the order isn't important.
 ADD COORDINATOR coordinatorId WITH CONFIG {"bolt_server": boltServer, "coordinator_server": coordinatorServer}; 
 ```
 
+<Callout type="info">
+
+If you are running `ADD COORDINATOR` Cypher queries on coordinator with --coordinator-id=1, you only need to add other two coordinator instances. For example, when using 3 
```

Comment:
Sorry, I feel like we're still missing something. Perhaps an opening statement along the lines of 
`We need to inform the leader coordinator of the existence of the 2 follower coordinators`.